### ISSUE TITLE
Cannot pip install

### ISSUE THREAD

**Issue by mesala** [sallamander/multi-gpu-keras-tf#1]
When cloning the repository, going to the top level directory (actually I tried to do it from several) and then type "pip install multi-gpu-keras-tf", I get this error: 
Collecting multi-gpu-keras-tf Could not find a version that satisfies the requirement multi-gpu-keras-tf (from versions: ) No matching distribution found for multi-gpu-keras-tf

I am working on an instance on google compute engine, with ubuntu 16 and python 3.5.

I noted a misspelling in one of the directories: mutli-gpu-keras-tf  should be MULTI not MUTLI, not sure if that's related to my inability to install it. Tried to rename it anyway and didn't work.

Any help? thanks

**Comment by sallamander** [comment]
Hmmm, I just cloned it and did the same with no issues. I'm on Ubuntu 14.04 and a Python 3.5.2 conda distribution. Given the error I'm wondering if throwing `version = 1.0` into the [setup.py](https://github.com/sallamander/multi-gpu-keras-tf/blob/master/mutli-gpu-keras-tf/setup.py#L8) would solve it. Can you try that?

**Comment by mesala** [comment]
Thanks. I think this issue can be closed. Issues were probably related to me messing things with conda environments.  I also realized I was looking for the library with the wrong name, I was expecting it to be named "multi-gpu-keras*" although it gets installed as "dist-keras-tf". 
To be honest, although I finally managed to install it, I wasn't sure how to import it on python, so I ended up just copy-pasting the main class (MultiGPUModel).
Thanks a lot.
